tests/e2e/Services/Sites/SitesCustomServerTest.php

```diff
@@ -65,6 +65,149 @@ public function testCreateSite(): void
         $this->cleanupSite($siteId);
     }
 
+    public function testVariables(): void
```

Comment:
Let's add 1 more test that includes deployments and E2E tests with router.

I would like to have a secret variable, and make sure that build and SSR can both still access it.

You can take inspiration for the test from here, where we do E2E test with router: https://github.com/appwrite/appwrite/blob/feat-spa/tests/e2e/Services/Sites/SitesCustomServerTest.php#L907